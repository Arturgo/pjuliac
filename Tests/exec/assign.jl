
x = true
y = x = 42
println(y)
println(x)

z = (x = 21) + (y = 34)
println(z)
