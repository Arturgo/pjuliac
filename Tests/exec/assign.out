42
42
55
